src/pages/[user]/[repo].astro

```diff
@@ -132,6 +132,7 @@ const initialDictModule = await import(
 const initialDict = {
   [preferredLocale]: initialDictModule.default
 }
+console.log(repoIndex.downloadLinks)
```

Comment:
Generally speaking, we shouldn't have console.log() statements in production code unless there's a compelling reason for it.

Reply:
I concur